pkg/projects/Microsoft.NETCore.Console/project.json

```diff
@@ -0,0 +1,51 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore": "5.0.1-rc3-24126-00",
+    "Microsoft.NETCore.ConsoleHost": "1.0.0-rc3-24126-00",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2-rc3-24126-00",
```

Comment:
This is a different meta-package, primarily used for some ad-hoc testing scenarios but no shipping scenarios, for my initial port I just ported it as is. For this case though I do expect it to directly reference CoreCLR package.